app/src/test/java/org/mozilla/fenix/components/metrics/GleanMetricsServiceTest.kt

```diff
@@ -85,4 +86,31 @@ class GleanMetricsServiceTest {
         assertEquals(1, events[0].extra!!.size)
         assertEquals("APP_ICON", events[0].extra!!["source"])
     }
+
+    @Test
+    fun `awesomebar events is correctly recorded`() {
```

Comment:
nit: `awesomebar events are correctly recorded`

Reply:
oops.  I'll fix that when I refactor.  thanks for catching it.